src/frontend/index.js

```diff
@@ -0,0 +1,50 @@
+import express from 'express'
+const app = express()
+const PORT = process.env.PORT || 3000
+import * as data from './data.js'
+
+app.listen(PORT, () => {
+    console.log(`listening on port ${PORT}`)
+})
+
+app.use(express.static('dist'))
+app.use(express.json())
+
+app.get("/evaluation", (req, res) => { res.json({ message: "Hello from server!" }) })
+
+app.get("/evaluation/teams", (req, res) => {
+    res.json(data.teams)
+})
+
+app.get("/evaluation/results/:id", (req, res) => {
```

Comment:
the request url is like 
`/api/v1/evaluation/results/?team=7c90d362-c443-40da-b428-fab41656dd0f`

i can delete `/api/v1/` which is prefixed for all api's if you wish

Reply:
updated all routes to start with `/api/v1`